backend/src/referentiels/controllers/referentiels-scoring.e2e-spec.ts

```diff
@@ -411,6 +412,36 @@ describe('Referentiels scoring routes', () => {
     );
   });
 
+  it(`Export du snapshot pour un utilisateur non authentifié`, async () => {
```

Comment:
Plutôt utiliser `test`, va savoir pourquoi Amandine avait eu des problèmes incompréhensibles qui ont été solutionnés en changeant de `it` à `test`.